android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt

```diff
@@ -401,29 +386,29 @@ class MusicModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaM
     fun reset(callback: Promise) {
         if (verifyServiceBoundOrReject(callback)) return
 
-        musicService.stop()
+        musicService.stopPlayer()
         callback.resolve(null)
     }
 
     @ReactMethod
     fun play(callback: Promise) {
-        if (verifyServiceBoundOrReject(callback)) return
+//    if (verifyServiceBoundOrReject(callback)) return
```

Comment:
blocking: why did we remove all of these checks? users might still try to perform an action before they call `setupPlayer` and it should throw.

Reply:
Explained above.